sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredential.java

```diff
@@ -9,34 +9,24 @@
 import java.util.List;
 
 /**
- * <p>The DefaultAzureCredential is appropriate for most scenarios where the application is intended to ultimately be
- * run in Azure. DefaultAzureCredential combines credentials that are commonly used to authenticate when deployed,
- * with credentials that are used to authenticate in a development environment. The DefaultAzureCredential will
- * attempt to authenticate via the following mechanisms in order.</p>
+ * <p>DefaultAzureCredential simplifies authentication while developing apps that deploy to Azure by combining credentials
+ * used in Azure hosting environments with credentials used in local development. In production, it's better to use something
+ * else. For more information, see <a href="https://aka.ms/azsdk/java/identity/credential-chains#usage-guidance-for-defaultazurecredential">Usage guidance for DefaultAzureCredential</a>.
  *
- * <img src="doc-files/DefaultAzureCredentialAuthFlow.png" alt="">
+ * <p>Attempts to authenticate with each of these credentials, in the following order, stopping when one provides a token:</p>
  *
  * <ol>
- * <li>{@link EnvironmentCredential} - The DefaultAzureCredential will read account information specified via
```

Comment:
This does add an extra layer for dev to unwrap from IDE to learn about the usage.
The idea previously was to provide the conceptual info needed to use the credential within the Javadocs..

Reply:
I'm fine with that. It forces the customer to read the linked doc.